test/integration/patients/worklist/worklist.js

```diff
@@ -2081,16 +2086,21 @@ context('worklist page', function() {
   });
 
   specify('filters sidebar - done states', function() {
+    localStorage.setItem(`done-last-thirty-days_11111_11111-${ STATE_VERSION }`, JSON.stringify({
+      id: 'done-last-thirty-days',
+      states: ['55555', '66666', '77777'],
+    }));
+
     cy
-      .routeFlows()
+      .routeActions()
       .routeFlow()
       .routeFlowActions()
       .routePatientByFlow()
       .routeDirectories()
       .visit('/worklist/done-last-thirty-days')
-      .wait('@routeFlows')
+      .wait('@routeActions')
       .then(() => {
-        const storage = JSON.parse(localStorage.getItem(`done-last-thirty-days_11111-${ STATE_VERSION }`));
+        const storage = JSON.parse(localStorage.getItem(`done-last-thirty-days_11111_11111-${ STATE_VERSION }`));
```

Comment:
ah interesting.. this was a result of the state being triggered twice I think that it didn't need to be prefilled.  Honestly though, we're not testing the defaults here, we're setting them..  so I think this check should be removed as well as the localstorage set on 2089 so that the defaults are tested here.

Reply:
Good call, I've got that removed in the last push.